cmd/run.go

```diff
@@ -50,12 +50,18 @@ goc run . [--buildflags] [--exec] [--arguments]
 		gocBuild.GoRunArguments = goRunArguments
 		defer gocBuild.Clean()
 
-		// only save services in memory
-		cover.DefaultStore = cover.NewMemoryStore()
+		server := cover.Server{
```

Comment:
`cover.Server` has a strong dependency on the `store` property，but current implement can not guarantee it's effectiveness. So it's better to define a method to init the `cover.Server` struct and also make it not exported, like：
```
func NewServer(xxx) （*server, error）{}
```
This way provides a chance to make sure the returned `*Server` struct is expected since we can do more check if needed.